gulpfile.js

```diff
@@ -1,11 +1,85 @@
-var gulp = require('gulp');
-var uglify = require('gulp-uglify');
-var rename = require('gulp-rename');
 
-gulp.task('build', function() {
- gulp.src('diver.js')
-     .pipe(gulp.dest('dist/'))
-     .pipe(rename('diver.min.js')) 
-     .pipe(uglify())
-     .pipe(gulp.dest('dist'))
+var
```

Comment:
Well the issue says nothing at all. However, adding gulp watch requires a refactoring in the project structure, sans which adding gulp watch is of no use. Because syncing is useful if u can compile and reload code on browser and see the changes. And, I have only added those bits in restructuring which is minimum needed for watch cycle. For example i excluded the support of test framework from watch issue fix which is an additional feature but not needed for gulp watch implemention. And the moment scripts are the side effects of implementing gulp watch which is needed.